service/Microsoft.Azure.Devices/AmqpServiceClient.cs

```diff
@@ -101,8 +101,10 @@ public SendingAmqpLink SendingLink
 
         public async override Task CloseAsync()
```

Comment:
Do you mean separating roles of "CloseAsync()` and `Dispose()`? (Do you want to remove `Dispose()` s from `CloseAsync()`?